components/SearchBar.vue

```diff
@@ -79,6 +80,28 @@ const selectedSpecialty: Ref<Specialty | String> = ref('')
 const selectedLocation: Ref<string> = ref('')
 const selectedLanguage: Ref<Locale | String> = ref('')
 
+const anyLocation = "----Any----"
+
+function createCityDisplayText() {
+    let cityDisplayTextObject: { [key: string]: number } = {}
```

Comment:
while this is an efficient storage option, it's not very clear what values your using......you could use an array of objects here with two fields to make it a bit more clear

example

```js
const cityDisplayItems = [
  {
    cityText: 'dallas',
    cityCount: 2
  },
 {
    cityText: 'tokyo',
    cityCount: 5
  }
]
```